reloj_checador_team03/Gemfile

```diff
@@ -16,6 +16,7 @@ gem 'webpacker', '~> 5.0'
 # Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'wdm', '>= 0.1.0' if Gem.win_platform?
```

Comment:
This gem seems to be more into the `development` bucket.

Right below in this file 😄 

I'd also suggest to commit this alone, and describe in your commit why are you adding this dependency to the project, since this does not look like something related to kaminario nor the